packages/repocop/src/index.ts

```diff
@@ -49,6 +51,21 @@ async function writeEvaluationTable(
 	console.log('Finished writing to table');
 }
 
+async function writeVulnerabilitiesTable(
+	vulnerabilities: repocop_vulnerabilities[],
```

Comment:
The `as unknown as repocop_vulnerabilities[]` had me a little confused on L136, but I think that's because the type there has the additional `repo_owner` property, so could be avoided by changing the signature here to:

```suggestion
	vulnerabilities: Array<RepocopVulnerability & { repo_owner: string }>,
```

So below it'd become

```typescript
const vulnerabilities = evaluationResults
	.flatMap((result) => result.vulnerabilities)
	.flatMap((vuln) => combineVulnWithOwners(vuln, repoOwners));
await writeVulnerabilitiesTable(vulnerabilities, prisma);
```